docs/source/index.rst

```diff
@@ -79,6 +79,58 @@ optimization *studies*.
     study = optuna.create_study()  # Create a new study.
     study.optimize(objective, n_trials=100)  # Invoke optimization of the objective function.
 
+Web Dashboard
+-------------
+
+`Optuna Dashboard <https://github.com/optuna/optuna-dashboard>`_ is a real-time web dashboard for Optuna.
+You can check the optimization history, hyperparameter importance, etc. in graphs and tables.
+You don't need to create a Python script to call `Optuna's visualization <https://optuna.readthedocs.io/en/stable/reference/visualization/index.html>`_ functions.
+Feature requests and bug reports are welcome!
+
+.. image:: https://user-images.githubusercontent.com/5564044/204975098-95c2cb8c-0fb5-4388-abc4-da32f56cb4e5.gif
+
+``optuna-dashboard`` can be installed via pip:
+
+.. code-block:: bash
+
+  $ pip install optuna-dashboard
```

Comment:
Pygments (which is used by Sphinx behind the scenes) recommends using `console` to highlight a shell session.
https://pygments.org/docs/terminal-sessions/

```suggestion
.. code-block:: console

   $ pip install optuna-dashboard
```

Reply:
Also I would suggest that it's more common to use 3 space characters as an indent for reStructuredText directives.

Reply:
Thank you for your suggestions 🙏